xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -11,38 +11,36 @@ import ObservableStore
 import Combine
 
 //  MARK: View
-struct DetailView: View {
+struct MemoEditorDetailView: View {
     /// Detail keeps a separate internal store for editor state that does not
     /// need to be surfaced in higher level views.
     ///
     /// This gives us a pretty big efficiency win, since keystrokes will only
     /// rerender this view, and not whole app view tree.
     @StateObject private var store = Store(
-        state: DetailModel(),
+        state: MemoEditorDetailModel(),
         action: .start,
         environment: AppEnvironment.default
     )
     /// Is this view presented? Used to detect when back button is pressed.
     /// We trigger an autosave when isPresented is false below.
     @Environment(\.isPresented) var isPresented
     @Environment(\.scenePhase) private var scenePhase: ScenePhase
-    /// State passed down from parent
-    var state: DetailOuterModel
-    var send: (DetailOuterAction) -> Void
+    /// Initialization state passed down from parent
+    var description: MemoEditorDetailDescription
+    /// An address to forward notifications (informational actions)
+    var notify: (MemoEditorDetailNotification) -> Void
```

Comment:
Static view state `state` and outer send function `send` become `description` and `notify` to prevent confusion with internal `store.state` and `store.send`.

`description` is what is pushed onto the detail navigation stack. It's a minimal struct that contains only the information needed for the detail to bootstrap its internal store (address, and not much else).

`notify` is a send function that we use to notify the outer view of certain lifecycle events or requests.